src/App.css

```diff
@@ -957,36 +959,32 @@ body,
   }
 }
 
-@media (max-width: 1000px) {
+
+/* iPad Mini - Landscape */
+/* Samsung Galaxy Tab S8 - Landscape */
+@media (max-width: 1040px) {
   .earth_orbits_container {
-    left: 27vw;
+    top: -10vh;
   }
-}
 
-@media (max-width: 750px) {
-  .splash_screen_body {
-    width: 90%;
-  }
```

Comment:
Several breakpoints were moved to the bottom or adjusted

I changed the breakpoint for 1170 - 1180 to capture the iPad Air Landscape
I added breakpoint 1040 to target iPadMini - Landscape and Samsung Tab S8
I changed the 500px breakpoint to 1000px to force the iPad Air Portrait (and other similar device widths) to use the vertical mobile device design.
 the 500px breakpoint still exists so all of the other (phone type) devices will still work the same.